### PR TITLE
migrate network modification list display

### PR DESCRIPTION
**Pull request by Mathieu-Deharbe**
This PR extracts some code and localisation from gridstudy in order to use it in : https://github.com/gridsuite/gridexplore-app/pull/538

Migrated to : https://github.com/gridsuite/commons-ui/pull/606

### CODE REVIEW

File: src/components/graph/util/use-modification-label-computer.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@
  */
```
Comment (etiennehomer) [inline src/components/graph/util/use-modification-label-computer.jsx:6]: File to remove ? This code is available in commons-ui

**Review by etiennehomer — CHANGES_REQUESTED** [review]

File: src/components/graph/util/use-modification-label-computer.jsx (hunk 1 above)
Comment (Mathieu-Deharbe) [reply src/components/graph/util/use-modification-label-computer.jsx]: I changed the links but forgot to delete it !  Thank you.

**Review by etiennehomer — APPROVED** [review]

File: src/components/app-wrapper.jsx [hunk 2]
```diff
@@ -48,6 +50,8 @@ import { BrowserRouter } from 'react-router-dom';
 import { Provider, useSelector } from 'react-redux';
 import messages_en from '../translations/en.json';
 import messages_fr from '../translations/fr.json';
+import exportParameters_en from '../translations/external/export-parameters-en.json';
+import exportParameters_fr from '../translations/external/export-parameters-fr.json';
 import network_modifications_locale_en from '../translations/dynamic/network-modifications-locale-en';
```
Comment (etiennehomer) [inline src/components/app-wrapper.jsx:55]: To remove ?

Comment (Mathieu-Deharbe) [reply src/components/app-wrapper.jsx]: Yes. There was a mixup during the merges and conflict resolution. I removed a few things including this.

**Review by ghazwarhili — APPROVED** [review]

**Pull request merged**